components/profile/AdminPanel.tsx

```diff
@@ -0,0 +1,157 @@
+import { Menu, Dropdown } from 'antd';
```

Comment:
<img width="1440" alt="Screen Shot 2020-06-17 at 6 44 20 PM" src="https://user-images.githubusercontent.com/35946905/84968113-981ad880-b0ca-11ea-9c22-f2850abb452e.png">
There are duplicated tabs on the left hand side. Remove the one on the bottom.